demo/site/src/blocks/CookieSafeYouTubeVideoBlock.tsx

```diff
@@ -0,0 +1,25 @@
+"use client";
+import { CookieSafe, PropsWithData, useCookieApi, YouTubeVideoBlock } from "@comet/cms-site";
+import { YouTubeVideoBlockData } from "@src/blocks.generated";
+import { CookieFallback } from "@src/components/common/CookieFallback";
+import { cookieIds } from "@src/util/cookieIds";
+import styled from "styled-components";
+
+const aspectRatio = "16x9";
+
+export const CookieSafeYouTubeVideoBlock = (props: PropsWithData<YouTubeVideoBlockData>) => {
```

Comment:
Shouldn't this be done in the library YouTubeVideoBlock?

Reply:
Yes, that's a follow-up task: AG-378